packages/terra-functional-testing/CHANGELOG.md

```diff
@@ -2,9 +2,15 @@
 
 ## Unreleased
 
-* Changed
+* Breaking changes
+  * Updated to pull in the seleniarm 4.10 docker images.
+
+* Changes
   * Updated `uuid` to `v8.2.0` for consistency across Terra repos.
 
+* Added
+  * Added `useHttps` to use a secure http connection when the `gridUrl` flag is also provided.
```

Comment:
What is the purpose of this change?

Reply:
This was in response to this request from Binwei: https://jira2.cerner.com/browse/ION-47621?focusedCommentId=12668119&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-12668119

I'll update our jira and pr to reflect that this was added in